tests/storage/snapshot.go

```diff
@@ -1053,8 +1053,9 @@ var _ = SIGDescribe("VirtualMachineSnapshot Tests", func() {
 				origStatus := ss.Status
 				// zero out the times to be able to compare after
 				clearConditionsTimestamps(origStatus.Conditions)
-				ss.Status = nil
-				ss, err = virtClient.VirtualMachineSnapshot(ss.Namespace).Update(context.Background(), ss, metav1.UpdateOptions{})
+				patchPayload, err := patch.New(patch.WithRemove("/status")).GeneratePayload()
```

Comment:
```
Replace update with patch in snapshot test 

Signed-off-by: Roni Kishner <rkishner@redhat.com>
```

In your commit message, please explain why you are making this change. What is wrong and fragile about calling `Update` here?